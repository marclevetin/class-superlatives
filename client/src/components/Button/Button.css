.button {
  float: center;
  color: #e74944;
}
