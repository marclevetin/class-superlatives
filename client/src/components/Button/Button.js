import React from "react";
import "./Button.css";

// The ...props means, spread all of the passed props onto this element
// That way we don't have to define them all individually
const Button = props => (
    <button className='btn btn-primary' onClick={props.handleClick}>
      {props.words}
    </button>
);

export default Button;
