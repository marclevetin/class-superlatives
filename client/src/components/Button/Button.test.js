import React from 'react';
import Button from './Button';
import renderer from 'react-test-renderer';

test('string', () => {
  const component = renderer.create(
    <Button />
  );

  // initial snapshot test
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
