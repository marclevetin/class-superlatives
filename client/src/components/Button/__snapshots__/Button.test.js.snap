// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`string 1`] = `
<div>
  <p
    className="text-center"
  >
    <button
      className="btn btn-primary"
      onClick={undefined}
    />
  </p>
</div>
`;
